lambda/lambda.ml

```diff
@@ -48,14 +48,18 @@ type primitive =
   | Psetglobal of Ident.t
   (* Operations on heap blocks *)
   | Pmakeblock of int * mutable_flag * block_shape
-  | Pfield of int
+  | Pfield of int * immediate_or_pointer * mutable_flag
```

Comment:
Just curious (please don't change anything!): I understand the `immediate_or_pointer` argument was useful when there was a read barrier, but is it used in the current design?

Reply:
That's correct. We don't use the `immediate_or_pointer` information in the current design.

Reply:
OK, thanks for the confirmation.